src/main.rs

```diff
@@ -25,6 +26,15 @@ enum Opt {
     /// Show a list of possible KEY_XXX values
     ListKeys,
 
+    /// Listen to a device for events
```

Comment:
Please expand this comment to describe it in a bit more detail. Maybe something like this?
```suggestion
    /// Listen to a device for events and print them out, to aid in identifying labels for your configuration file.
```